### ISSUE TITLE
Bonus Round Animation 

### ISSUE THREAD

**Issue by andrefarrugia96** [RyanDelia/IMTAssignment2016#6]
In the bonus round will you be implementing any type of animation to the objects falling ?

**Comment by RyanDelia** [comment]
Yes i am going to implement the animation that when the objects collide with the cooler they will make some sort of animated feature i as thinking of doing a +1 animated icon on collision.
